config/development.json

```diff
@@ -205,8 +205,6 @@
 		"wordpress-action-search": false,
 		"wpcom-user-bootstrap": false,
 		"newsletter/stats": true,
-		"newsletter/paid-subscribers": true,
-		"woa-logging": true,
-		"woa-logging-moved": true
+		"newsletter/paid-subscribers": true
```

Comment:
Please keep this list in alphabetical order (and other config files in `/config`) as per the [README file here](https://github.com/Automattic/wp-calypso/blob/dbbed47493fca010895952775bd0852868e7d23e/config/README.md#feature-flags):

> Please make sure to add new feature flags alphabetically so they are easy to find

🙏

Reply:
Tidied in 674a4fdd3b14ffbe28c96948f2dd9b65521ddeef CC @JuanLucha

Reply:
Awesome, appreciate it! 🙇